docs/src/main/asciidoc/Bootable_Guide.adoc

```diff
@@ -0,0 +1,197 @@
+[[Bootable_Guide]]
+= Bootable JAR Guide
+WildFly team;
+:revnumber: {version}
+:revdate: {localdate}
+:toc: macro
+:toclevels: 3
+:toc-title: Bootable JAR Guide
+:doctype: book
+:icons: font
+:source-highlighter: coderay
+ifdef::env-github[:imagesdir: images/]
+
+// ifndef::ebook-format[:leveloffset: 1]
+
+(C) 2020 The original authors.
+
+ifdef::basebackend-html[toc::[]]
+:numbered:
+
+[[wildfly_bootable_JAR_development]]
+= WildFly bootable JAR application development
+
+This document details the steps to follow in order to develop a WildFly application
+packaged as a bootable JAR. A bootable JAR can be run both on bare-metal and cloud platforms.
+
+This document contains the minimal information set required to build and run a WildFly bootable JAR. 
+Complete information on the Maven plugin usage can be found in the bootable JAR https://docs.wildfly.org/bootablejar/[documentation].
+
+Developing an application packaged as a bootable JAR is not different from developing an application for WildFly server.
+The extra steps required to package your application inside a bootable JAR are handled by the 
+https://github.com/wildfly-extras/wildfly-jar-maven-plugin[_org.wildfly.plugins:wildfly-jar-maven-plugin_] Maven plugin.
+
+
+[[wildfly-jar-maven-plugin-pom-configuration]]
+== Adding the bootable JAR Maven plugin to your pom file
+
+Add the plugin to your Maven build:
+
+[source,xml]
+----
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.wildfly.plugins</groupId>
+      <artifactId>wildfly-jar-maven-plugin</artifactId>
+      <configuration>
+        ...
+      </configuration>
+      <executions>
+        <execution>
+          <goals>
+            <goal>package</goal>
+          </goals>
+        </execution>
+      </executions>
+    </plugin>
+  </plugins>
+</build>
+----
+
+The link:#wildfly-jar-maven-plugin-galleon-configuration[next chapter] covers the plugin configuration items that are required to identify the WildFly server version and content.
+
+[[wildfly-jar-maven-plugin-galleon-configuration]]
+== Galleon configuration
+
+The Bootable JAR Maven plugin depends on https://docs.wildfly.org/galleon/[Galleon] to construct the WildFly server contained in the JAR file. 
+
+Galleon is configured thanks to the Maven plugin ```<configuration>``` element.
+
+The first required piece of information that Galleon needs is a reference to the WildFly Galleon feature-pack. 
+WildFly Galleon feature-pack is a maven artifact (a zipped file that contains the server content). 
+This feature-pack, as well as the feature-packs on which its depends,
+are deployed in public maven repositories. 
+
+When the bootable JAR Maven plugin builds a JAR, WildFly feature-packs are retrieved and 
+their content is assembled to create the server contained in the JAR.
+
+Once you have identified a WildFly Galleon feature-pack, you need to select a set of 
+link:#wildfly_defined_galleon_layers[WildFly Galleon layers] that are used to compose the server.
```

Comment:
Are these "WildFly Galleon Layers" or "WildFly Layers"?  The target heading drops "Galleon" from the title.

Reply:
That is WildFly Layers, I fixed those.